README.md

```diff
@@ -1,21 +1,21 @@
-# Shibboleth IdP Version 4 Installer
+# Shibboleth IdP Version 5 Installer
 
 ## Overview
-The AAF Shibboleth IdP Installer is designed to automate the install of version 4 for the [Shibboleth IdP](https://shibboleth.atlassian.net/wiki/spaces/IDP4/overview) on a dedicated with one of the following supported operating systems;
+The AAF Shibboleth IdP Installer is designed to automate the install of version 5 for the [Shibboleth IdP](https://shibboleth.atlassian.net/wiki/spaces/IDP5/overview) on a dedicated server with one of the following supported operating systems;
 * Rocky Linux 8 or 9
-* CentOS 7, Stream 8 or Stream 9
-* RedHAT 7, 8 or 9
-* ORACLE Linux 7, 8 or 9
+* Stream 8 or Stream 9
+* Redhat 8 or 9
+* ORACLE Linux 8 or 9
 * Ubuntu 20.04 (Focal Fossa) or 22.04 (Jammy Jellyfish)
 
-For a full set of documentation and guidance on upgrading from Shibboleth version 3 please refer to the [AAF IdPv4 Installer Knowledge base](https://aaf.freshdesk.com/support/solutions/articles/19000120020-shibboleth-idpv4-installer).
+For a full set of documentation and guidance on upgrading from Shibboleth version 4 please refer to the [AAF IdPv4 Installer Knowledge base](https://support.aaf.edu.au/support/solutions/articles/19000159910-shibboleth-idp-version-5-installer).
 
 ## Status
 This software is actively being developed and maintained. It is ready for use in production environments.
 
 This version now enables the technical connection to eduGAIN, the global federation of federations.
 
-This release is for Shibboleth version 4.3.1 running on Jetty 9.4.51
+This release is for Shibboleth version 5.1.3 running on Jetty 12.0.12
```

Comment:
Just FYI: Jetty has a newer release, 12.0.13 coming soon :-)
There's a bug (Scott raised it - JSP temp directory regression, possibly due to fix for #12044 #12124) in 12.0.12

Reply:
Will update when it becomes available.